packages/widget/src/state/wallets.ts

```diff
@@ -22,12 +34,13 @@ export type MinimalWallet = {
       index?: number;
       sourceChainID?: string;
     };
-  }) => Promise<string | undefined>;
+  }) => Promise<{ address: string | undefined; logo?: string }>;
```

Comment:
returning logo from getAddress is so that we can show the accurate logo for wallet connect in the recovery address / destination address in SwapExecutionPage